src/map/buyingstore.cpp

```diff
@@ -798,6 +798,20 @@ static int buyingstore_autotrader_free(DBKey key, DBData *data, va_list ap) {
 	return 0;
 }
 
+/**
+* Update buyer location
+* @param sd: Player's session data
+*/
+void buyingstore_update_buyer_location(map_session_data &sd)
```

Comment:
```suggestion
void buyingstore_update(map_session_data &sd)
```
After your last commit it is not only location that is getting updated.